_projects/authenticated_3rd_party_invites.md

```diff
@@ -0,0 +1,104 @@
+---
+name: 3rd Party Authorised Room Membership
+desc: Invite and kick users in a room based on some external authorisation. 
+requirements:
+ - Understanding of the matrix Client-Server API
+ - Familiarity with Javascript & React
+ - Familiarity with a programming language with a good matrix SDK (for the
+   server side component).
+difficulty: medium
+issues:
+mentors:
+ - Cadair
+ - Half-Shot
+initiatives:
+ - GSoC
+tags:
```

Comment:
Can you also add either a `175h` or `350h` tag depending on how long you think the project should take?

Reply:
I don't think the element knocking one had this either?